responsibleai/vision/components/rai_vision_insights/spec.yaml

```diff
@@ -1,54 +1,65 @@
 $schema: http://azureml/sdk-2-0/CommandComponent.json
 name: rai_vision_insights
 display_name: RAI Vision Insights
-version: 0.0.7
+version: 0.0.8
 tags:
   Preview: ""
 type: command
 inputs:
   task_type:
     type: string
     enum: ['image_classification', 'multilabel_image_classification', 'object_detection']
-  data_folder:
-    type: string 
-    optional: true
+    description: The type of task to perform
   model_input: # mlflow model name:version
     type: mlflow_model
     optional: false
+    description: The model to be used for computing insights
   model_info:
     type: string # model name:version
     optional: false
+    description: The model to be used for computing insights
   test_dataset:
     type: mltable
+    description: The test dataset to be used for computing insights
   target_column_name:
     type: string
+    description: The target column name
   maximum_rows_for_test_dataset:
     type: integer
     default: 5000
+    description: The maximum number of rows to use from the test dataset
   classes:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The class names for the target column
```

Comment:
Maybe reword to "The list of class names.."

Reply:
done!